cfg/examples/suction_fc_policy.yaml

```diff
@@ -0,0 +1,34 @@
+policy:
+  type: fully_conv_suction
+
+  sampling_method: top_k
+  gqcnn_stride: 4
+  gqcnn_recep_h: 96
+  gqcnn_recep_w: 96
+
+  # filtering params
+  max_grasps_to_filter: 50
+  filter_grasps: 0
+
+  # metrics
+  metric:
+    type: fcgqcnn
+    gqcnn_model: /path/to/your/FC-GQ-Image-Wise-Suction
```

Comment:
Is this supposed to be here? What is the difference between this and the other fc suction config?

Reply:
If you mean the difference between this and cfg/examples/replication/dex-net_4.0_fc_suction.yaml, I was under the impression that the policy configs under cfg/examples/replication were for *exact* replication whereas the ones just under cfg/examples were just general ones. I guess the only real difference is that the exact ones point to a downloaded model. This might just have been an artifact of a merge-we can remove these then.